pkg/httphelper/httphelper.go

```diff
@@ -6,14 +6,30 @@ import (
 	"net/http"
 )
 
-func Error(w http.ResponseWriter, err error) {
+type JSONError struct {
+	Code    int         `json:"code"`
```

Comment:
I was thinking that `code` should be a string that can be used as a static identifier to compare against when error handling. Examples:

```text
syntax_error
object_exists
not_found
```

What I'm not sure about is whether we need to namespace them based on the component and then have generic codes as well.